event_parsing.go

```diff
@@ -0,0 +1,135 @@
+package gitlab
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+const eventTypeHeader = "X-Gitlab-Event"
+
+// webhookEventType returns the event type for the given request.
+func webhookEventType(r *http.Request) string {
```

Comment:
I think this should be public?

Reply:
Oops! Overly aggressive symbol renaming.